lib/utils/workspace-utils.ts

```diff
@@ -76,6 +76,17 @@ export async function saveWorkspace({
   return { data: { workspace: data, repos: repoData, contributors: contributorsData }, error };
 }
 
+export async function upgradeWorkspace({ workspaceId, sessionToken }: { workspaceId: string; sessionToken: string }) {
+  const { data, error } = await fetchApiData<{ sessionId: string }>({
+    path: `auth/checkout/workspaces/${workspaceId}/session`,
+    method: "POST",
+    body: {},
```

Comment:
If this isn't getting added automatically by the empty post request, a header here for `Content-Length: 0` would be ideal